### DISCUSSION TITLE
Moment Coefficients from ADflow not matching with actual value

### DISCUSSION THREAD

**Discussion by vishnu67sankar** [mdolab/adflow#288] [Q&A] [answered]
Hello there!

I was running a 2D airfoil analysis based on the test case02 from the "[AIAA Aerodynamic Design Optimization Discussion Group](https://drive.google.com/file/d/13uJ_KxpPqGuCBs5KxCy08EUcz71wqXWE/view)". The results for which are available in the paper [here](http://oddjob.utias.utoronto.ca/dwz/Miscellaneous/Christopher_Lee_Aerodynamic_Shape_Optimization_of_Benchmark_Problems_Using_Jetstream_revised.pdf), as published by the authors, on page number 14, table 7. I generated a mesh using pyHyp, and ran the analysis in ADflow. The Cl and Cd values I got are about 0.8708 and 222.2 counts respectively, which are roughly quite close to the values given in the paper above. However, the moment coefficient I got from ADflow is way off. I have attached the output file below which monitors the residuals, Cd, Cl, and Cm values throughout the simulation. For completeness, the output contains the moment coefficient about the X, Y and Z axis. But none of them is close to the actual value (~ -0.0948). I have checked the orientation of my airfoil, it is in the XY plane with the chord lying along the X-axis. I have uploaded the mesh and other required files [here ](https://drive.google.com/drive/folders/1F_0NyNOuK0vQhXsLCP5uzZ7nRKyeL8KE?usp=sharing) incase anyone wants to have a look at it. 

Does anyone know what's going on with the moment coefficient calculation? Am I missing something here?

[output_analysis.txt](https://github.com/mdolab/adflow/files/11699533/output_analysis.txt)
[adflow_parameters.txt](https://github.com/mdolab/adflow/files/11699554/adflow_parameters.txt)

**Comment by eirikurj** [comment, marked as answer]
Hey, for the aeroproblem, did you set the reference point, i.e., `xRef`? See https://mdolab-baseclasses.readthedocs-hosted.com/en/latest/pyAero_problem.html for details.

**Reply by vishnu67sankar** [reply]
Hey! Thanks for the catch, I had used `xref` instead of `xRef`, no error was raised (not sure if a warning did) and it totally slipped my mind to check the right case. Thanks a lot for your prompt reply! The issue is solved now with `xRef`.
